api/base/serializers.py

```diff
@@ -202,6 +202,19 @@ def should_be_none(self, instance):
         return not isinstance(self.field, RelationshipField)
 
 
+class HideIfDraft(ConditionalField):
```

Comment:
Minor: Would be better as "HideIfDraftRegistration" unless we want to use this for other draft types. If that's the case, we should generalize or add a todo to generalize.

Reply:
🆗